examples/gps_datalogging.py

```diff
@@ -29,14 +29,24 @@
 # lines to import the necessary library and initialize the SD card:
 # NOT for use with a single board computer like Raspberry Pi!
```

Comment:
Lines 28-30 seem to have an incorrect comment, since even before the new changes for sdcardio, the lines were already uncommented.

(I could not mark line 28 in this comment -- it's one above the boundary of changed lines.)

Reply:
Sorry, what specifically about it is incorrect? Isn't the whole block supposed to be in a block quote?

Reply:
It says
```py
# If writing to SD card on a microcontroller customize and uncomment these
# lines to import the necessary library and initialize the SD card:
# NOT for use with a single board computer like Raspberry Pi!
```
But the lines were already uncommented (you added alternatives, but still, one set is uncommented.

Reply:
Oh, I thought those comments were referring to the whole block being commented out with triple quotes.

Reply:
That being said, I feel like it'd be better to have something that just detects if it's running on blinka and then if it isn't does all the SD card stuff

Reply:
Oh, I did not see the surrounding triple-quotes, sorry. I was expecting to see `#` comments. Maybe replace the triple-quotes with '###` or `##` or something to set all the `"""`-enclosed lines off. Otherwise I don't think it is obvious that there is commented-out coder there.

Reply:
> That being said, I feel like it'd be better to have something that just detects if it's running on blinka and then if it isn't does all the SD card stuff

You could also do logging to CIRCUITPY, not the SD card, so it's not just blinka.

I was thinking of something like:

```py
try:
    import sdcardio
    ...
except ImportError:
    import adafruit_sdcard
    ...
```

Reply:
Since this is just an example, maybe assume there is an SD card, or split into two examples, or something.

Reply:
I think it's a good idea to have it logging to the SD card since a lot of the GPS modules have SD card slots. I agree that it would be a good idea to just assume it's on CIRCUITPY and say to comment it out if you're on a Raspberry Pi